src/main/java/adris/altoclef/Settings.java

```diff
@@ -283,6 +283,18 @@ public class Settings implements IFailableConfigFile {
      */
     private String idleCommand = "";
 
+
+    /**
+     * If set, will run this command after death.
+     * <p>
+     * For example, try setting this to "@goto <Your base coords>" to make the bot go to the base before continuing the task it was given.
+     * Or setting it to "I died!" will send the i died message
+     * Or setting it to "/back" will execute /back on the server
+     * When running altoclef commands, if the prefix is not @, the set prefix should be used for the command and not @
```

Comment:
Maybe also add a note that `{deathmessage}` will be replaced with the death message from the screen (with a quick example of what a death message looks like)

Reply:
that's funny, i thought i added that